frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -60,14 +66,32 @@ pods:
           KAFKA_HEAP_OPTS: "-Xms{{BROKER_JAVA_HEAP}}M -Xmx{{BROKER_JAVA_HEAP}}M"
         goal: RUNNING
         cmd: >
+          ./bootstrap -resolve=false &&
```

Comment:
@benclarkwood is there any downside to having this as the first line in the command? This means that we would not template the correct values of `JAVA_HOME`?

Reply:
Yeah that's def possible, and IIRC that's why we've commonly set `JAVA_HOME` first.